packages/gasket-cli/package.json

```diff
@@ -190,21 +174,5 @@
   },
   "engines": {
     "node": ">= 8.0.0"
-  },
-  "oclif": {
-    "commands": "./src/commands",
-    "bin": "gasket",
-    "plugins": [
-      "@oclif/plugin-help",
-      "@oclif/plugin-not-found",
-      "@oclif/plugin-warn-if-update-available"
-    ],
-    "hooks": {
-      "init": "./src/hooks/init"
-    },
-    "warn-if-update-available": {
```

Comment:
Can you add a follow-up ticket for a custom warn-if-update-available check? It seems to encourage devs to keep current.